File: Examen Practico 1er Parcial/Ejercicio 12/TrianguloPascal.cpp
#include <iostream>
using namespace std;

int main() {
    int n, i, j, k, resultado;

    cout << "Ingrese el tamano del Triangulo de Pascal: ";
    cin >> n;

    for (i = 0; i < n; i++) {

        for (j = 0; j < n - i - 1; j++) {
            cout << " ";
        }

        for (j = 0; j <= i; j++) {
            resultado = 1;
            for (k = 1; k <= j; k++) {
                resultado = resultado * (i - k + 1) / k;
            }
            cout << resultado << "   ";
        }

        cout << endl;
    }

    return 0;
}

File: Examen Practico 1er Parcial/triangulo_pascal.f
      program triangulo_pascal
      integer n,i,j,k,c,resultado

      print *, 'Ingrese el tamaño del Triangulo de Pascal:'
      read *, n

      do i = 0, n-1

          do j = 1, n-i-1
              write(*,'(A)', advance='no') ' '
          end do

          do j = 0, i
              resultado = 1
              do c = 1, j
                  resultado = resultado * (i - c + 1) / c
              end do
              write(*,'(I5)', advance='no') resultado
          end do

          write(*,*)

      end do
      print*,"presione Enter para salir"
      read(*,*)
      end program triangulo_pascal

